redundant-bmc/src/rbmc_tool.cpp

```diff
@@ -219,27 +264,72 @@ sdbusplus::async::task<> displaySiblingBMCInfo(sdbusplus::async::context& ctx,
         auto bmcState = BMCState::convertBMCStateToString(state);
         bmcState = bmcState.substr(bmcState.find_last_of('.') + 1);
 
-        printParam("Redundancy Enabled:", rProps.redundancy_enabled);
-        printParam("Failovers Allowed:", rProps.failovers_allowed);
-        printParam("BMC State:", bmcState);
-        printParam("FW Version Hash:", fwVersion);
-        printParam("Provisioned:", true); // TODO
+        output["Redundancy Enabled"] = rProps.redundancy_enabled;
+        output["Failovers Allowed"] = rProps.failovers_allowed;
+        output["BMC State"] = bmcState;
+        output["FW Version Hash"] = fwVersion;
+        output["Provisioned"] = true; // TODO
     }
     catch (const sdbusplus::exception_t& e)
```

Comment:
in the getLocalBMCInfo function it's just catching a std::exception, so I should probably do the same here.